app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: int) -> None:
```

Comment:
```suggestion
            brand: int
        ) -> None:
```
it's better to format it like this